tests/openebs-pool-container-failure_test.go

```diff
@@ -86,6 +88,8 @@ var _ = Describe("BDD of openebs pool container failure experiment", func() {
 	restartCountSumBefore := 0
 	cspPodLabels := "app=cstor-pool"
 	cspPodNs := "openebs"
+	chaosNamespace := "litmus"
```

Comment:
this is fixed for all exp, so can you please create a separate type.go file and put all the variables there, refer `chaos-operator` repo. Do the same thing for all files